examples/aa-simple-dapp/src/hooks/useAlchemyProvider.ts

```diff
@@ -64,11 +67,9 @@ export const useAlchemyProvider = () => {
 
       switch (type) {
         case PluginType.SESSION_KEY:
-          return provider
-            .extend(SessionKeyPlugin.providerMethods)
-            .installSessionKeyPlugin({
-              args: [[]],
-            });
+          return sessionKeyPlugin?.installSessionKeyPlugin({
```

Comment:
I think we should revisit returning `null` if the hook is not connected in a later PR. I think the hook should probably only be used in connected contexts because it makes it much easier to work with. Otherwise, you have to do this null check everywhere you want to use it 🤷